src/pages/Beta/Pool/PoolImportModal/PoolImport/index.tsx

```diff
@@ -30,7 +31,23 @@ const PoolImport = ({ currency0, currency1, openTokenDrawer, setActiveField, onM
   const { t } = useTranslation()
   const theme = useContext(ThemeContext)
 
-  const [pairState, pair] = usePair(currency0 ?? undefined, currency1 ?? undefined)
+  const chainId = useChainId()
+
+  let unwrapcurrency0 = wrappedCurrency(currency0, chainId)
+  let unwrapcurrency1 = wrappedCurrency(currency1, chainId)
+
+  if (currency0 && !(currency0 instanceof Currency)) {
+    const _currency0 = currency0 as any
```

Comment:
can we use `as Token` here ?

Reply:
I'm not sure about that, I will to test and if work I change to `as Token`